server/app/views/admin/programs/ProgramBlockView.java

```diff
@@ -13,22 +14,30 @@
 
 abstract class ProgramBlockView extends BaseHtmlView {
   /** Renders a div with internal/admin program information. */
-  protected final DivTag renderProgramInfo(ProgramDefinition programDefinition) {
-    DivTag programTitle =
-        div(programDefinition.adminName()).withId("program-title").withClasses("text-3xl", "pb-3");
-    DivTag programDescription = div(programDefinition.adminDescription()).withClasses("text-sm");
+  protected final DivTag renderProgramInfo(ProgramDefinition program) {
+    DivTag title =
+        div(program.localizedName().getDefault())
```

Comment:
None of the tests exercise this change, can you update the tests so they do?  Presumably they are setting the visible and admin name to the same hyphenated thing?

Reply:
Updated unit tests. For browser tests it's going to be a bit more work so I decided to make it into a follow up PR. Currently browser tests always use the same hyphenated string for public and admin names.